### DISCUSSION TITLE
出现这个是什么原因的

### DISCUSSION THREAD

**Discussion by koi-pig** [dreamhunter2333/cloudflare_temp_email#500] [General]
![image](https://github.com/user-attachments/assets/5a47c200-2541-4473-8163-597b4a0de80d)

**Comment by dreamhunter2333** [comment]
感谢反馈，后台地址配置正确么，可以打开 f12 看下

**Comment by koi-pig** [comment]
![image](https://github.com/user-attachments/assets/87cedc48-f4c3-4336-bee2-d14f1dcd6888)
好像好了但是出来新问题这个发送报400，我没开启那个发送验证那个功能啊

**Comment by dreamhunter2333** [comment]
提示是没启用 resend 或者 添加加已验证地址

**Comment by shaoyouvip** [comment]
> 提示是没启用 resend 或者 添加加已验证地址

401 Need User Token 是怎么了 用是可以正常能用。

**Comment by koi-pig** [comment]
验证已经关了，
![image](https://github.com/user-attachments/assets/ebd5ea38-0c75-47b2-999c-ec7dd3b15f8e)
ji记录已经添加到cf里去了，但是这个地方还是失败的

**Comment by dreamhunter2333** [comment]
> > 提示是没启用 resend 或者 添加加已验证地址
> 
> 401 Need User Token 是怎么了 用是可以正常能用。

会话过期了，需要重新登录

**Comment by dreamhunter2333** [comment]
> 验证已经关了， ![image](https://private-user-images.githubusercontent.com/99023123/389868302-ebd5ea38-0c75-47b2-999c-ec7dd3b15f8e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI2MTk1MTEsIm5iZiI6MTczMjYxOTIxMSwicGF0aCI6Ii85OTAyMzEyMy8zODk4NjgzMDItZWJkNWVhMzgtMGM3NS00N2IyLTk5OWMtZWM3ZGQzYjE1ZjhlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTI2VDExMDY1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEwYzY2OGFmOWVlMmJkZTM4MWFlYTZlMjBmNTQ4YmRlOWIxMjBmMTNmYjE2ZGMzODkyZDA2NWJkODlkMjQ0OGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.WbdH6da6FmbzatjFAcvVzmhp6saR64OKvJgSIz487pQ) ji记录已经添加到cf里去了，但是这个地方还是失败的

dig 命令是否能查到呢

**Comment by koi-pig** [comment]
查不到啊，但是已经添加了，有添加resend 添加成功的吗
